src/utils/og.ts

```diff
@@ -0,0 +1,30 @@
+import type { MetaInfo } from 'vue-meta'
+import type { MetaPropertyName } from 'vue-meta/types/vue-meta'
+
+export const createDetailPageMeta = (title?: string, thumbnail?: string) => {
```

Comment:
Refactoring suggestion: I think the arguments should be a dictionary mapping OG name with content as in type `Record<string, string>`.

That way the `meta` array can be generated by iterating over the items of the dict.

Reply:
I think it would be better to add this change when we add any more computed `meta` properties. Now, we set the `title` separately from the `meta` array, so it's not simply the iteration over the dict items.